vnext/JSI/Shared/NapiJsiV8RuntimeHolder.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "NapiJsiV8RuntimeHolder.h"
+
+// V8-JSI
+#include <V8JsiRuntime.h>
+
+using namespace facebook::jsi;
+using namespace facebook::react;
+
+using std::shared_ptr;
+using std::unique_ptr;
+
+namespace Microsoft::JSI {
+
+// See napi_ext_schedule_task_callback definition.
+/*static*/ void NapiJsiV8RuntimeHolder::ScheduleTaskCallback(
+    napi_env env,
```

Comment:
>napi_env [](http://example.com/codeflow?start=4&length=8)

(nit) what are these structures? should they be moved? &&?

Reply:
This is part of the NAPI. It is a pointer. It cannot be moved.